assets/src/sass/content/_search-results.scss

```diff
@@ -0,0 +1,33 @@
+// Group all style customizations for the 2019-2 transparency report together.
+.search-results {
+	&__tabs {
+		border-bottom: 1px solid #ccc;
+		margin-bottom: 2rem !important;
+		padding-bottom: 0.35rem !important;
+		padding-top: 2rem !important;
+
+		a {
+			padding: 0.5rem 1rem;
+			margin-right: 0.35rem;
+			border: 1px solid #ccc;
```

Comment:
Could we use [theme variables](https://github.com/wikimedia/shiro-wordpress-theme/blob/main/assets/src/sass/helpers/_variables.scss) like `$color-ltgray` or `$wmui-color-base70` for the colors in these styles?

Reply:
Definitely!